### ISSUE TITLE
Issue loading SadlBuiltinFunctions.sadl for test cases

### ISSUE THREAD

**Issue by crapo** [crapo/sadlos2#227]
I created the class ISadlBuiltinFunctionsModelContentProvider to provide the SadlBuiltinFunctions implicit model to test cases, designed after ISadlImplicitModelContentProvider but simpler as there are no contributions from other sources. I modified AbstractSadlTest to load this implicit model. What I don't understand is why I had to comment out line 74 to cause the Resource to be loaded. This can be tested with the test cases in com.ge.research.sadl.tests.model.SadlModelProcessorTestTypeChecking.

**Comment by kittaakos** [comment]
It should be:
```
diff --git a/sadl3/com.ge.research.sadl.parent/com.ge.research.sadl.tests/src/com/ge/research/sadl/tests/AbstractSadlTest.xtend b/sadl3/com.ge.research.sadl.parent/com.ge.research.sadl.tests/src/com/ge/research/sadl/tests/AbstractSadlTest.xtend
index b477cc6..50c279e 100644
--- a/sadl3/com.ge.research.sadl.parent/com.ge.research.sadl.tests/src/com/ge/research/sadl/tests/AbstractSadlTest.xtend
+++ b/sadl3/com.ge.research.sadl.parent/com.ge.research.sadl.tests/src/com/ge/research/sadl/tests/AbstractSadlTest.xtend
@@ -58,23 +58,23 @@
 
 	@Inject protected ParseHelper<SadlModel> parseHelper;
 	@Inject protected Provider<XtextResourceSet> resourceSetProvider;
-	@Inject protected ISadlImplicitModelContentProvider modelContentProvider;
+	@Inject protected ISadlImplicitModelContentProvider implicitModelContentProvider;
 	@Inject protected ISadlBuiltinFunctionsModelContentProvider builtinModelContentProvider;
 	
 	@Accessors(PROTECTED_GETTER)
 	XtextResourceSet currentResourceSet;
 	
 	private val Supplier<Void> implicitModelSupplier = Suppliers.memoize[
-		val uri = URI.createURI(SadlConstants.SADL_IMPLICIT_MODEL_SYNTHETIC_URI);
-		if (!currentResourceSet.resources.map[uri.lastSegment].exists[it == SadlConstants.SADL_IMPLICIT_MODEL_FILENAME]) {
-			val resource = loadResource(modelContentProvider.content, uri);
+		val implicitModelUri = URI.createURI(SadlConstants.SADL_IMPLICIT_MODEL_SYNTHETIC_URI);
+		if (!currentResourceSet.resources.map[URI.lastSegment].exists[it == SadlConstants.SADL_IMPLICIT_MODEL_FILENAME]) {
+			val resource = loadResource(implicitModelContentProvider.content, implicitModelUri);
 			OntModelProvider.find(resource)
 		}
-		val bfuri = URI.createURI(SadlConstants.SADL_BUILTIN_FUNCTIONS_SYNTHETIC_URI);
-//		if (!currentResourceSet.resources.map[bfuri.lastSegment].exists[it == SadlConstants.SADL_BUILTIN_FUNCTIONS_FILENAME]) {
-			val resource = loadResource(builtinModelContentProvider.content, bfuri);
+		val builtinFunctionsUri = URI.createURI(SadlConstants.SADL_BUILTIN_FUNCTIONS_SYNTHETIC_URI);
+		if (!currentResourceSet.resources.map[URI.lastSegment].exists[it == SadlConstants.SADL_BUILTIN_FUNCTIONS_FILENAME]) {
+			val resource = loadResource(builtinModelContentProvider.content, builtinFunctionsUri);
 			OntModelProvider.find(resource)
-//		}
+		}
 		
 		return null;
 	]

```

**Comment by kittaakos** [comment]
Will be fixed as part of #226.

**Comment by kittaakos** [comment]
PR: https://github.com/crapo/sadlos2/pull/234.
